docs/encyclopedia/temporal-nexus.mdx

```diff
@@ -46,3 +48,13 @@ For example, in Temporal Cloud when you execute a Nexus Operation in a caller Wo
 This means you don’t have to use Nexus RPC directly, only the Temporal SDK along with the Temporal Service.
 
 Temporal Nexus is available both in self-hosted deployments and Temporal Cloud.
+
```

Comment:
Please turn this into a tip admonition or resources list. We generally don't have the Related Read Containers outside of the Evaluate section.

Reply:
fixed. note I did see a lot of `RelatedReadContainers` in the encyclopedia section, but they're not used for TOC.